src/ducks/loan/LoanDetailsPage.jsx

```diff
@@ -10,10 +11,11 @@ import cx from 'classnames'
 import styles from 'ducks/loan/LoanDetailsPage.styl'
 import LoanProgress from 'ducks/loan/LoanProgress'
 import { Padded, Wrapper } from 'components/Spacing'
+import { flowRight as compose } from 'lodash'
 
 const DATE_FORMAT = 'DD/MM/YY'
 
-const Row = translate()(props => {
+const DumbRow = translate()(props => {
```

Comment:
I always use _ as suffix when my component is not yet enhanced but Dumb is better 👍 

Careful: It seems DumbRow get `translate()`ed twice ;)